src/scripts/main.js

```diff
@@ -1,7 +1,28 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const div = document.createElement('div');
+
+  div.classList.add('notification', type);
```

Comment:
It's a good practice to check if the 'type' argument is one of the expected values ('success', 'error', 'warning') before adding it as a class to the div. This can prevent unexpected behavior if the function is called with an incorrect type.